### PR TITLE
release: our series is 16

### PR DESCRIPTION
**Pull request by mvo5**

### CODE REVIEW

**Comment by zyga** [comment]
+1

**Comment by pedronis** [comment]
LGTM, later we may want to retweak the release interface but this looks right to me for now

File: store/snap_remote_repo_test.go [hunk 1]
```diff
@@ -163,7 +163,7 @@ func (t *remoteRepoTestSuite) TestUbuntuStoreRepositoryHeaders(c *C) {
 
 	t.store.applyUbuntuStoreHeaders(req, "", nil)
 
-	c.Assert(req.Header.Get("X-Ubuntu-Release"), Equals, release.String())
+	c.Assert(req.Header.Get("X-Ubuntu-Release"), Equals, release.Get().Series)
```
Comment (niemeyer) [inline store/snap_remote_repo_test.go]: Can we please have a string here? Right now it is testing implementation against implementation, which will pass even if we break it.

**Comment by niemeyer** [comment]
LGTM assuming the commented line actually says "16".

**Pull request merged**